docs/version-overrides.md

```diff
@@ -44,3 +66,5 @@ This document lists the version overrides for vulnerable (nested) dependencies a
 :warning: Attention :warning: 
 * `trim`, `trim-newlines` and `glob-parent` are dependencies of `storybook` that is used in `@sap-ux/ui-components`. Once a new version without the vulnerable dependency is available, it is to be used and this override can be removed.
 * `semver` is used by too many modules. Override can be removed or kept for specific dependencies after we have cleaned up our devDependencies.
+* `tough-cookie` no fix available yet for jest-environment-jsdom
+*  `word-wrap` no fix available yet for @typescript-eslint/* modules.
```

Comment:
Fixed with fork https://www.npmjs.com/package/@aashutoshrathi/word-wrap  version 1.2.5+

Reply:
Fork is no longer required. (released 1 week ago - original finding is near 3 weeks old)
There is an official fix now https://github.com/jonschlinkert/word-wrap/releases/tag/1.2.5
Note that the 1.2.5 version should be used to not get space changes.